{
    "id": "CVE-2020-15701",
    "product": "apport",
    "version": "2.20.1 ",
    "vulnerability": [
        "CWE-755 Improper Handling of Exceptional Conditions"
    ],
    "description": "An unhandled exception in check_ignored() in apport/report.py can be exploited by a local attacker to cause a denial of service. If the mtime attribute is a string value in apport-ignore.xml, it will trigger an unhandled exception, resulting in a crash. Fixed in 2.20.1-0ubuntu2.24, 2.20.9-0ubuntu7.16, 2.20.11-0ubuntu27.6.",
    "poc": {
        "reference": [
            "No PoCs from references."
        ],
        "github": [
            "https://github.com/Live-Hack-CVE/CVE-2020-15701"
        ]
    }
}